docker/tests/test-fedora-binaries.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import subprocess
+import sys
+
+binaries = [
+    "tezos-client",
+    "tezos-admin-client",
+    "tezos-node",
+    "tezos-signer",
+    "tezos-codec",
+    "tezos-baker-PtMumbai",
+    "tezos-accuser-PtMumbai",
+    "tezos-smart-rollup-client-PtMumbai",
+    "tezos-smart-rollup-node-PtMumbai",
+    "tezos-baker-PtNairob",
+    "tezos-accuser-PtNairob",
+    "tezos-smart-rollup-client-PtNairob",
+    "tezos-smart-rollup-node-PtNairob",
+]
+
+
+def test(cleanup: bool):
+    processed_binaries = []
+    for binary in binaries:
+        try:
+            install_command = f"sudo dnf install -y {binary}"
+            installation_result = subprocess.run(install_command, shell=True)
+            assert installation_result.returncode == 0
+
+            processed_binaries.append(binary)
+
+            check_binary_command = f"sudo {binary} --version"
```

Comment:
```suggestion
            check_binary_command = f"{binary.replace('tezos', 'octez')} --version"
```

Reply:
using `tezos-` prefixed binaries results in warning message, which pollutes the output (same on ubuntu)